crates/language-tests/src/tests/schema/mod.rs

```diff
@@ -325,7 +334,13 @@ impl<'de> Deserialize<'de> for SurrealRecordId {
 		D: serde::Deserializer<'de>,
 	{
 		let source = String::deserialize(deserializer)?;
-		let v = syn::value(&source).map_err(<D::Error as serde::de::Error>::custom)?;
+		let capabilities = CoreCapabilities::all().with_experimental(Targets::All);
+		let res = syn::value_with_capabilities(&source, &capabilities);
+		if let Err(ref e) = res {
+			println!("Failed to parse rid: {source}");
+			println!("Error: {e}");
+		};
```

Comment:
Why are you printing this here? Any error happening here will be raised to the command line already.